thoth/metrics_exporter/jobs.py

```diff
@@ -41,7 +41,18 @@
 _LOGGER = logging.getLogger("thoth.metrics_exporter.jobs")
 
 
-def countGraphSyncJobStatus(job_list_items: list) -> dict:
+def get_namespaces(environment_variables: list) -> list:
```

Comment:
I think there is no need to have this parametrized, and, if you return a set it should automatically handle our test environment where all `*_NAMESPACE` configuration is set to `thoth-test-core` (so only one item in the set will be present).